### PR TITLE
spotify_player 0.14.1 (new formula)

### PR DESCRIPTION
**Pull request by tresni**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Comment by tresni** [comment]
Is there anything else I need to do here? I think this just needs reviewed at this point.

**Comment by tresni** [comment]
squashed to a single commit as that seems to be the process based on other merged formulas.

File: Formula/spotify_player.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  uses_from_macos "expect" => :test
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "alsa-lib"
+    depends_on "dbus"
+  end
+
+  def install
+    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix
```
Comment (chenrui333) [inline Formula/spotify_player.rb:19]:
```suggestion
    # Ensure that the `openssl` crate picks up the intended library.
    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix
    ENV["OPENSSL_NO_VENDOR"] = "1"
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
bump to the latest release 0.14.1

**Review by p-linnane — APPROVED** [review]

**Pull request merged**